### PR TITLE
ce_3_1 with Contour

### PR DESCRIPTION
**Pull request by doqhne**
Creates NCL example ce_3_1 using geocat-viz Contour function. Includes link to previous version that does not use Contour.

Resolves issue #467 

[Previous script](https://geocat-examples.readthedocs.io/en/v2022.5.0/gallery/Contours/NCL_ce_3_1.html#sphx-glr-gallery-contours-ncl-ce-3-1-py)

Original NCL plot:
https://www.ncl.ucar.edu/Applications/Images/ce_3_1_lg.png

New version of plot using Contour:
ce_3_1_python

### CODE REVIEW

**Review by anissa111 — APPROVED** [review]

**Comment by anissa111** [comment]
Wait I missed something here. Let me do a quick edit and then I'll merge

**Comment by anissa111** [comment]
@hCraker @michaelavs I've introduced an updated "warning" message for experimental scripts here. We'll need to update other scripts to match, but I suggest doing that in a dedicated PR.

File: Gallery/Contours/NCL_ce_3_1.py [hunk 1]
```diff
@@ -4,27 +4,27 @@
...
-   - Comparing styles of map tickmarks labels
-   - Changing the stride of the colorbar labels
    - Zooming in on a particular area on the map
-   - Turning off the addition of a longitude cyclic point
 
 See following URLs to see the reproduced NCL plot & script:
     - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/ce_3.ncl
     - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/ce_3_1_lg.png
+
+Note:
```
Comment (hCraker) [inline Gallery/Contours/NCL_ce_3_1.py:15]:
```suggestion
.. Warning::
```
If you use this header, you can make a nicely formatting warning in an orange box. It's really hard to miss and would be good to use here.

File: Gallery/Contours/NCL_ce_3_1.py [hunk 2]
```diff
@@ -4,27 +4,27 @@
...
+    This is an experimental version of the script. Go to the [latest](https://geocat-examples.readthedocs.io/en/latest/gallery/Contours/NCL_ce_3_1.html) version for the stable version.
+
+    See the geocat-viz experimental documentation for the functionality in this script [here](https://geocat-viz.readthedocs.io/en/experimental/).
```
Comment (hCraker) [inline Gallery/Contours/NCL_ce_3_1.py:18]:
```suggestion
    This is an experimental version of the script. Go to the `latest <https://geocat-examples.readthedocs.io/en/latest/gallery/Contours/NCL_ce_3_1.html>`_ version for the stable version.

    See the geocat-viz experimental documentation for the functionality in this script `here <https://geocat-viz.readthedocs.io/en/experimental/>`_.
```

Fix for link formatting

**Review by hCraker — CHANGES_REQUESTED** [review]
Some of the links aren't formatted right (Sphinx uses a strange syntax rather than markdown), and I recommended a way to make a nicely formatted and highly visible warning on the website. Once these suggestions are merged, I think this is good to go!

**Review by hCraker — APPROVED** [review]

**Comment by anissa111** [comment]
> Some of the links aren't formatted right (Sphinx uses a strange syntax rather than markdown), and I recommended a way to make a nicely formatted and highly visible warning on the website. Once these suggestions are merged, I think this is good to go!

Ah I forgot about that, thanks for the catch @hCraker!

**Pull request merged**